src/schema/rules/tabular_data/microephys.yaml

```diff
@@ -0,0 +1,99 @@
+microephysProbes:
+  selectors:
+    - datatype == ["ecephys", "icephys"]
+    - suffix == "probes"
+    - extension == ".tsv"
+  initial_columns:
+    - probe_id
+    - type__probes
+    - x__probes
+    - y__probes
+    - z__probes
+  columns:
+    probe_id: required
+    type__probes: required
+    x__probes: recommended
+    y__probes: recommended
+    z__probes: recommended
+    manufacturer__probes: recommended
+    device_serial_number: recommended
+    electrode_count: optional
+    width__probes: optional
+    height__probes: optional
+    depth__probes: optional
+    dimension_unit: optional
+    alpha_rotation: recommended
```

Comment:
@ree-gupta can you please delete this so it shows the latest changes google docs?